pyramid_nacl_session/session.py

```diff
@@ -1,15 +1,19 @@
+# -*- coding: utf-8 -*-
+
 from pyramid.session import (
     BaseCookieSessionFactory,
     PickleSerializer,
 )
 
 from .serializer import EncryptedSerializer
 
+
+# pylint: disable=too-many-arguments,invalid-name
```

Comment:
I don't believe we currently use pylint on this repository. Please do not add extra linter information.